### PR TITLE
Migrate to Python 3

### PR DESCRIPTION
**Pull request by thii**
Fixes https://github.com/bazelbuild/tulsi/issues/278.

### CODE REVIEW

File: src/TulsiGenerator/Scripts/bazel_build.py [hunk 1]
```diff
@@ -831,7 +830,7 @@ def ReaderThread(file_handle, out_buffer):
     process = subprocess.Popen(command,
                                stdout=subprocess.PIPE,
                                stderr=subprocess.STDOUT,
-                               bufsize=1)
```
Comment (keith) [inline src/TulsiGenerator/Scripts/bazel_build.py:834]: why did this one change? doesn't look like the behavior changed between versions?

**Review by keith — APPROVED** [review]

File: src/TulsiGenerator/Scripts/bazel_build.py (hunk 1 above)
Comment (thii) [reply src/TulsiGenerator/Scripts/bazel_build.py:834]: To workaround this warning:

```
subprocess.py:848: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
```

Comment (DavidGoldman) [reply src/TulsiGenerator/Scripts/bazel_build.py:834]: bufsize=1 requires universal_newlines=True. We're also working on this internally

**Comment by thii** [comment]
Taking the upstream changes instead https://github.com/bazelbuild/tulsi/pull/299

**Pull request closed without merging**